cmd/stack_validate.go

```diff
@@ -78,6 +79,10 @@ Runs the following validation tests against the stack and its templates:
 			stackName := filepath.Base(stackPath)
 			rootConfig.Info.Log("stackName is: ", stackName)
 
+			if imageRegistry != "dev.local" && imageNamespace == "appsody" {
```

Comment:
Is this necessary?

Reply:
I thought it would make more sense to say "if you specify a registry, you must specify the namespace". I will remove this